downstream/modules/builder/con-pip-based-req.adoc

```diff
@@ -0,0 +1,11 @@
+[id="con-pip-based-req"]
+
+= Note for `pip`-based requirements
+
+Python requirements files are combined into a single file using the `requirements-parser` library in order to support complex syntax. Entries from separate collections that give the same package name will be combined into the same entry, with the constraints combined.
+
+See https://github.com/ansible/awx-ee/blob/devel/_build/requirements_combined.txt[this link] for an example of a combined Python requirements file.
```

Comment:
We might just want to remove this sentence, since there is no longer a viewable example of a combined requirements file.